src/frontend/src/components/CreateGallery.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import Image from "next/image";
+// import GalleryItem from './GalleryItem';
+import { IoIosArrowBack, IoIosArrowForward, IoIosAdd } from "react-icons/io";
+import { HiOutlinePlusSm } from "react-icons/hi";
+import { GrLocation } from "react-icons/gr";
+
+const CreateGallery = () => {
+  return (
+    <div className={`mx-auto mt-20 rounded-lg shadow-[rgba(50,50,93,0.25)_0px_6px_12px_-2px,_rgba(0,0,0,0.3)_0px_3px_7px_-3px]`} style={{ maxWidth: '70vw', height: '72vh', backgroundColor: 'white', display: 'flex', alignItems: 'center', justifyContent: 'space-between'}}>
+      <div className='left-section'>
+        <div className="top-left-section flex items-center ml-12">
```

Comment:
Consider remove "top-left-section" if it doesn't serve any functional purpose to help keeping our codebase cleaner and more maintainable.